rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -39,7 +41,7 @@
             ),
             13000L,
             (BridgeMethodExecutorTyped) Bridge::addFederatorPublicKey,
-            blockchainConfig -> !blockchainConfig.isRskip123(),
+            blockchainConfig -> !blockchainConfig.isActive(RSKIP123),
```

Comment:
I think we could rename this parameter to something like `executionBlockActivations`